[ADD] l10n_in_withholding: add TDS/TCS threshold alert (India) (#11109)
odoo/documentation: content/applications/finance/fiscal_localizations/india.rst

   .. tab:: TCS

      - Navigate to **Accounting --> Configuration --> Chart of Accounts**. open the desired account, and configure the TDS/TCS section.
         .. image:: india/tcs-2061h-coa1.png
            :alt: TCS 206(1H) configuration with CoA
         .. image:: india/tcs-2061h-coa2.png
            :alt: TCS 206(1H) configuration with CoA
      - Based on the COA used during the recording of the Customer Invoice, the TCS threshold limit will be checked.
         .. image:: india/tcs-coa-invoice-config.png
            :alt: configuration CoA with Vendor Bill
      - At the time of recording a Customer Invoice, if the limit specified in the TCS section configured in COA is exceeded, the system will advise you to apply TCS.
         .. image:: india/tcs-alert.gif
            :alt: TCS Alert
      - The advice will disappear once you apply TCS.

      **TCS Advise**

      - The TCS advice will be visible on the Customer Invoice, indicating the applicable section under which TCS may be applied.
         .. image:: india/tcs-warning.png
            :alt: TCS Advise

      **How to apply TCS?**

      - TCS is directly applicable in the tax on invoice lines.

> .. note::
>    - In Odoo, the aggregate total is calculated for partners sharing the same PAN number, across all branches of the company.
>    - **Example:**
>       .. list-table::
>          :header-rows: 1
>          :widths: 10 20 10 20 15
> 
>          * - **Branch**
>            - **Customer**
>            - **Invoice**
>            - **Transaction Amount (₹)**
>            - **PAN Number**
>          * - IN - MH
>            - XYZ Enterprise - GJ
>            - Invoice 1
>            - ₹50,000
>            - ABCPX1234E
>          * - IN - MH
>            - XYZ Enterprise - GJ
>            - Invoice 2
>            - ₹30,000
>            - ABCPX1234E
>          * - IN - MH
>            - XYZ Enterprise - MH
>            - Invoice 3
>            - ₹40,000
>            - ABCPX1234E
>          * - IN - DL
>            - XYZ Enterprise - GJ
>            - Invoice 4
>            - ₹20,000
>            - ABCPX1234E
>          * - IN - GJ
>            - XYZ Enterprise - MH
>            - Invoice 5
>            - ₹60,000
>            - ABCPX1234E
> 
>       - **Aggregate Total** = 50,000 + 30,000 + 40,000 + 20,000 + 60,000 = ₹200,000
>       - The aggregate total for all customers (XYZ Enterprise - GJ, MH, DL) sharing the PAN number ABCPX1234E across all branches is ₹200,000.

Reviewer: ```suggestion
In Odoo, the aggregate total is calculated for partners sharing the same PAN number, across all branches of the company.
.. example::
   .. list-table::
      :header-rows: 1
      :widths: 10 20 10 20 15
      * - **Branch**
        - **Customer**
        - **Invoice**
        - **Transaction Amount (₹)**
        - **PAN Number**
      * - IN - MH
        - XYZ Enterprise - GJ
        - Invoice 1
        - ₹50,000
        - ABCPX1234E
      * - IN - MH
        - XYZ Enterprise - GJ
        - Invoice 2
        - ₹30,000
        - ABCPX1234E
      * - IN - MH
        - XYZ Enterprise - MH
        - Invoice 3
        - ₹40,000
        - ABCPX1234E
      * - IN - DL
        - XYZ Enterprise - GJ
        - Invoice 4
        - ₹20,000
        - ABCPX1234E
      * - IN - GJ
        - XYZ Enterprise - MH
        - Invoice 5
        - ₹60,000
        - ABCPX1234E

**Aggregate total** = 50,000 + 30,000 + 40,000 + 20,000 + 60,000 = ₹200,000
The aggregate total for all customers (XYZ Enterprise - GJ, MH, DL) sharing the PAN number
ABCPX1234E across all branches is ₹200,000.
```

>    - The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these limits, go to **Accounting --> Configuration --> Taxes** Advance options Tab and Sections.
>       .. image:: india/tds-tcs-section-modifiy.png
>          :alt: TDS/TCS section modify

Reviewer: ```suggestion
.. note::
   The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these
   limits, go to :menuselection:`Accounting --> Configuration --> Taxes`. In the :guilabel:`Advanced
   Options` tab, click on the  :icon:`fa-arrow-right` :guilabel:`(internal link)` icon of the
   :guilabel:`Section` field.
      .. image:: india/tds-tcs-section-modifiy.png
         :alt: TDS/TCS section modify
```
I also recommend renaming the image so that "modify" is spelled correctly.

      :alt: TDS/TCS Settings

#. **How to configure TDS/TCS Section on COA?**

.. tabs::

   .. tab:: TDS

      - Navigate to **Accounting --> Configuration --> Chart of Accounts**. open the desired account, and configure the TDS/TCS section.
         .. image:: india/tds-194c-coa.png
            :alt: TDS 194C configuration with CoA
         .. image:: india/tds-194J-coa.png
            :alt: TDS 194J configuration with CoA

      - Based on the COA used during the recording of the vendor bill, the TDS threshold limit will be checked.
         .. image:: india/tds-coa-vendor-bill-config.png
            :alt: configuration CoA with Vendor Bill

      - At the time of recording a Vendor Bill, if the limit specified in the TDS section configured in COA is exceeded, the system will advise you to apply TDS.
         .. image:: india/tds-alert.gif
            :alt: TDS Alert
      - The advice will disappear once you apply TDS.

      **TDS Advise**

      - The TDS advice will be visible on the Vendor Bill, indicating the applicable section under which TDS may be applied.
         .. image:: india/tds-warning.png
            :alt: TDS Advise

      **How to apply TDS?**

>       - To apply TDS, click on the **"TDS Entry"** button available on the Vendor Bill. A popup window will appear where you can specify the TDS details. Confirm the entry to apply the TDS.

Reviewer 2: ```suggestion
      - To apply TDS, click on the **"TDS Entry"** button available on the Vendor Bill / Payment. A popup window will appear where you can specify the TDS details. Confirm the entry to apply the TDS.
```
Not show GIF just show photo of popup

   .. tab:: TCS

      - Navigate to **Accounting --> Configuration --> Chart of Accounts**. open the desired account, and configure the TDS/TCS section.
         .. image:: india/tcs-2061h-coa1.png
            :alt: TCS 206(1H) configuration with CoA
         .. image:: india/tcs-2061h-coa2.png
            :alt: TCS 206(1H) configuration with CoA
      - Based on the COA used during the recording of the Customer Invoice, the TCS threshold limit will be checked.
         .. image:: india/tcs-coa-invoice-config.png
            :alt: configuration CoA with Vendor Bill
      - At the time of recording a Customer Invoice, if the limit specified in the TCS section configured in COA is exceeded, the system will advise you to apply TCS.
         .. image:: india/tcs-alert.gif
            :alt: TCS Alert
      - The advice will disappear once you apply TCS.

      **TCS Advise**

      - The TCS advice will be visible on the Customer Invoice, indicating the applicable section under which TCS may be applied.
         .. image:: india/tcs-warning.png
            :alt: TCS Advise

      **How to apply TCS?**

      - TCS is directly applicable in the tax on invoice lines.

>          .. image:: india/tcs-apply.gif

Reviewer 2: No need to add a screenshot for this.
The user know it.

   - **Example:**
      .. list-table::
         :header-rows: 1
         :widths: 10 20 10 20 15

         * - **Branch**
           - **Customer**
           - **Invoice**
           - **Transaction Amount (₹)**
           - **PAN Number**
         * - IN - MH
           - XYZ Enterprise - GJ
           - Invoice 1
           - ₹50,000
           - ABCPX1234E
         * - IN - MH
           - XYZ Enterprise - GJ
           - Invoice 2
           - ₹30,000
           - ABCPX1234E
         * - IN - MH
           - XYZ Enterprise - MH
           - Invoice 3
           - ₹40,000
           - ABCPX1234E
         * - IN - DL
           - XYZ Enterprise - GJ
           - Invoice 4
           - ₹20,000
           - ABCPX1234E
         * - IN - GJ
           - XYZ Enterprise - MH
           - Invoice 5
           - ₹60,000
           - ABCPX1234E

      - **Aggregate Total** = 50,000 + 30,000 + 40,000 + 20,000 + 60,000 = ₹200,000
      - The aggregate total for all customers (XYZ Enterprise - GJ, MH, DL) sharing the PAN number ABCPX1234E across all branches is ₹200,000.

>    - The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these limits, follow the process below:

Reviewer 2: ```suggestion
   - The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these limits go to **Accounting --> Configuration --> Taxes** Advance options Tab and Sections**
```
just show screenshot not gif

   .. image:: india/profit-and-loss-report.png
      :alt: Profit and Loss report

.. _india/tds-tcs-threshold:

TDS/TCS threshold alert
=======================

> :guilabel:`TDS (Tax Deducted at Source)` and :guilabel:`TCS (Tax Collected at Source)` are tax
> provisions under Indian law, triggered when transaction amounts exceed specified thresholds.
> This alert notifies users when the value of invoices or bills surpasses these limits, prompting
> the application of the appropriate TDS/TCS.

Reviewer: ```suggestion
:abbr:`TDS (tax deducted at source)` and :abbrl:`TCS (tax collected at source)` are tax provisions
under Indian law, triggered when transaction amounts exceed specified thresholds. This alert
notifies users when the value of invoices or bills surpasses these limits, prompting the application
of the appropriate TDS/TCS.
```
Sorry if I was unclear about the GUI labels, but it should be clear in the documentation guidelines - GUI labels are just for when you're instructing the user to do something with an interactive element within Odoo, like clicking a smart button or completing a field. In that case, the text that you see in Odoo is what would get the GUI label. Let's use the abbr markup for these and follow the guidelines on capitalization.

> To configure Odoo to advise you on when to apply TDS/TCS, simply set the :guilabel:`TDS/TCS section`

Reviewer 3: ```suggestion
To configure Odoo to advise you on when to apply TDS/TCS, set the :guilabel:`TDS/TCS section`
```
Not necessary

.. note::
   The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these
   limits, go to :menuselection:`Accounting --> Configuration --> Taxes`. In the :guilabel:`Advanced
   Options` tab, click on the  :icon:`fa-arrow-right` :guilabel:`(internal link)` icon of the
   :guilabel:`Section` field.

   .. image:: india/tds-tcs-section-modify.png
      :alt: TDS/TCS section modify

Applying TCS/TDS on invoices and bills
--------------------------------------

Based on the account used on the customer invoice or vendor bill, Odoo checks the TCS/TDS threshold
limit. If the limit specified in the :guilabel:`TCS/TDS Section` of the account is exceeded, Odoo
displays an alert that suggests applying the appropriate TCS/TDS. The alert will disappear once the
TCS/TDS is applied.

.. image:: india/tcs-warning.png
   :alt: TCS advice

**TCS** is directly applicable in the tax on the invoice lines. To apply **TDS**, click the
:guilabel:`TDS Entry` smart button on the vendor bill/payment. The popup window allows specifying
the TDS details. Confirm the entry to apply the TDS.

.. image:: india/tds-apply.png
   :alt: TDS application

> In Odoo, the aggregate total is calculated for partners sharing the same PAN number, across all branches of the company.

Reviewer 3: ```suggestion
In Odoo, the aggregate total is calculated for partners sharing the same PAN number across all company branches.
```

        - **Invoice**
        - **Transaction Amount (₹)**
        - **PAN Number**
      * - IN - MH
        - XYZ Enterprise - GJ
        - Invoice 1
        - ₹50,000
        - ABCPX1234E
      * - IN - MH
        - XYZ Enterprise - GJ
        - Invoice 2
        - ₹30,000
        - ABCPX1234E
      * - IN - MH
        - XYZ Enterprise - MH
        - Invoice 3
        - ₹40,000
        - ABCPX1234E
      * - IN - DL
        - XYZ Enterprise - GJ
        - Invoice 4
        - ₹20,000
        - ABCPX1234E
      * - IN - GJ
        - XYZ Enterprise - MH
        - Invoice 5
        - ₹60,000
        - ABCPX1234E

-  **Aggregate total** = 50,000 + 30,000 + 40,000 + 20,000 + 60,000 = ₹200,000
-  The aggregate total for all customers (XYZ Enterprise - GJ, MH, DL) sharing the PAN number

>    ABCPX1234E across all branches is ₹200,000.

Reviewer: My mistake on this edit - can you put these lines within the example block and remove the numbers? They don't need to be part of a list

.. note::
   The TDS/TCS sections are pre-configured with threshold limits. If you need to modify these
   limits, go to :menuselection:`Accounting --> Configuration --> Taxes`. In the :guilabel:`Advanced
   Options` tab, click on the  :icon:`fa-arrow-right` :guilabel:`(internal link)` icon of the
   :guilabel:`Section` field.

   .. image:: india/tds-tcs-section-modify.png

>       :alt: TDS/TCS section modify

Reviewer: One last thing, let's actually put this at the bottom of the configuration section, before the "applying TCS/TDS on invoices and bills" section.
